sys/src/9/amd64/qmalloc.c

```diff
@@ -136,7 +136,7 @@ static Pool pimagmem = {
 	.quantum=	32,
 	.alloc=		malloc,
 	.merge=		sbrkmerge,
-	.flags=		0,
+	.flags=		POOL_PARANOIA | POOL_VERBOSITY | POOL_DEBUGGING | POOL_LOGGING,
```

Comment:
This should not be the default, so you're going to have to remove it from this commit.